components/VM_Arm/src/main.c

```diff
@@ -927,15 +933,38 @@ static int load_linux(vm_t *vm, const char *kernel_name, const char *dtb_name, c
     return 0;
 }
 
-void parse_camkes_linux_attributes(void)
+static int parse_camkes_linux_attributes(void)
 {
     linux_ram_base = strtoul(linux_address_config.linux_ram_base, NULL, 0);
     linux_ram_paddr_base = strtoul(linux_address_config.linux_ram_paddr_base, NULL, 0);
     linux_ram_size = strtoul(linux_address_config.linux_ram_size, NULL, 0);
     linux_ram_offset = strtoul(linux_address_config.linux_ram_offset, NULL, 0);
     dtb_addr = strtoul(linux_address_config.dtb_addr, NULL, 0);
+
+    if (strlen(linux_image_config.initrd_name)) {
+        if (!strlen(linux_address_config.initrd_max_size)) {
+            ZF_LOGE("initrd_name is specified, initrd_max_size is not");
```

Comment:
Do we really need `initrd_max_size`. It feels like an optional parameter to me that is unused in simple cases.

Reply:
as far as I can tell, initrd_max_size is completely unused